.evergreen/generate_evergreen_tasks.js

```diff
@@ -564,7 +564,7 @@ oneOffFuncAsTasks.push({
 });
 
 oneOffFuncAsTasks.push({
-  name: 'run-custom-csfle-tests-master',
+  name: 'run-custom-csfle-tests-master-mongocryptd',
```

Comment:
```suggestion
  name: 'run-custom-csfle-tests-mongocryptd-master',
```

non-blocking - this is just for consistency with the shared library tests.